src/assisted_test_infra/test_infra/controllers/node_controllers/nutanix_controller.py

```diff
@@ -125,6 +125,13 @@ def restart_node(self, node_name: str) -> None:
         vm.power_on()
 
     def get_ingress_and_api_vips(self, is_highly_available: bool = True):
+        if not self._entity_config.vip_dhcp_allocation:
+            if not self._entity_config.api_vip:
+                raise ConnectionError("API VIP is not set")
```

Comment:
shouldn't it be ``raise ValueError`` if it's a configuration mismatch made by the user?
or maybe i just don't fully understand the case here

Reply:
Right, `ValueError` fits better here